ydb/docs/en/core/reference/configuration/security_config.md

```diff
@@ -0,0 +1,211 @@
+# Section `security_config`
+
+The `domains_config.security_config` section defines [authentication](../../security/authentication.md) modes, initial configuration of local [users](../../concepts/glossary.md#access-user) and [groups](../../concepts/glossary.md#access-group), and their [access rights](../../concepts/glossary.md#access-right).
+
+```yaml
+domains_config:
+  ...
+  security_config:
+    # authentication mode configuration
+    enforce_user_token_requirement: false
+    enforce_user_token_check_requirement: false
+    default_user_sids: <authentication token for anonymous requests>
+    all_authenticated_users: <group name for all authenticated users>
+    all_users_group: <group name for all users>
+
+    # initial security configuration
+    default_users: <default user list>
+    default_groups: <default group list>
+    default_access: <default access rights on the cluster scheme root>
+
+    # access list configuration
+    viewer_allowed_sids: <list of SIDs that are allowed to view the cluster state>
+    monitoring_allowed_sids: <list of SIDs that are allowed to monitor and change the cluster state>
+    administration_allowed_sids: <list of SIDs that are allowed cluster administration>
+```
+
+[//]: # (TODO: wait for pull/9387, dynamic_node_registration to add info about "register_dynamic_node_allowed_sids: <список SID'ов с правами подключения динамических нод в кластер>")
+
+## Configuring authentication mode {#security-auth}
+
+#|
+|| Parameter | Description ||
+|| `enforce_user_token_requirement` | Enforces user [authentication](../../security/authentication.md) mode.
+
+- `enforce_user_token_requirement: true` — User authentication is mandatory. Requests to {{ ydb-short-name }} must include an [auth token](../../concepts/glossary.md#auth-token).
+
+    Requests to {{ ydb-short-name }} undergo authentication and authorization.
+
+- `enforce_user_token_requirement: false` — User authentication is optional. Requests to {{ ydb-short-name }} are not required to include an [auth token](../../concepts/glossary.md#auth-token).
+
+    Requests without an auth token are processed in [anonymous mode](../../security/authentication.md#anonymous) without authorization.
+
+    Requests with an auth token undergo authentication and authorization. But if an authentication error occurs, requests are still processed in anonymous mode.
+
+    When `enforce_user_token_check_requirement: true`, requests with authentication errors are blocked.
+
+[//]: # (TODO: добавить про ошибки проверки права на доступ к базе данных, когда появится место для ссылки)
+
+If the `default_user_sids` parameter is defined and not empty (see the description below), its value is used instead of the missing auth token. In this case, authentication and authorization is performed for the [access subject](../../concepts/glossary.md#access-subject) defined in `default_user_sids`.
```

Comment:
```suggestion
If the `default_user_sids` parameter is defined and not empty (see the description below), its value is used instead of the missing auth token. In this case, authentication and authorization are performed for the [access subject](../../concepts/glossary.md#access-subject) defined in `default_user_sids`.
```